src/brands/peapods/lib/PeaConnections.js

```diff
@@ -36,27 +68,145 @@ const PeaConnections = ({ classes, followers, followings, tags, groups }) => {
     <Grid container>
-      {rows.map(({ title, data }) =>
-        data.length ? (
-          <Grid container direction="column">
-            <Grid item>
-              <Typography className={classes.categoryHeading}>
-                {title}
-              </Typography>
-            </Grid>
-
-            <Grid container spacing={2}>
-              {data.map(item => (
-                <Grid key={item.name} item>
-                  <PeaSocialAvatar {...item} />
+      <Grid container alignItems="center" justify="space-between">
+        <Typography className={classes.heading}>Connections</Typography>
+        {loading ? (
+          <PeaLoadingSpinner size={20} />
+        ) : (
+          <IconButton
+            aria-describedby={settingsPopoverId}
+            onClick={onConnectionsSetting}
+          >
+            <PeaIcon
+              icon={'settings'}
+              color="secondary"
+              bgColor={'white'}
+              size={'small'}
+              shadow={false}
+            />
+          </IconButton>
+        )}
+      </Grid>
+
+      {!loading ? (
+        <>
+          {rows.map(({ title, data }) =>
+            data.length ? (
+              <Grid container direction="column">
+                <Grid item>
+                  <Typography className={classes.categoryHeading}>
+                    {title}
+                  </Typography>
                 </Grid>
-              ))}
-            </Grid>
+
+                <Grid container spacing={2}>
+                  {data.map(item => (
+                    <Grid key={item.name} item>
+                      <a
+                        href={`https://${item.social}.com/${item.unique}`}
+                        target="_blank"
+                        rel="noreferrer noopener"
+                      >
+                        <PeaSocialAvatar {...item} />
+                      </a>
+                    </Grid>
+                  ))}
+                </Grid>
+              </Grid>
+            ) : null,
+          )}
+        </>
+      ) : null}
+
+      <Popover
+        id={settingsPopoverId}
+        open={settingsOpen}
+        anchorEl={anchorEl}
+        onClose={onSettingsClose}
+        anchorOrigin={{
+          vertical: 'center',
+          horizontal: 'right',
+        }}
+        transformOrigin={{
+          vertical: 'center',
+          horizontal: 'right',
+        }}
+        PaperProps={{
+          style: {
+            minHeight: 100,
+            minWidth: 310,
+            padding: 12,
+          },
+        }}
+      >
+        <Grid container direction="column">
+          <Grid container alignItems="center" justify="space-between">
+            <Typography className={classes.settingsHeading}>
+              {connect ? 'Connect Account' : 'Disconnect Account'}
+            </Typography>
+            <PeaSwitch
+              checked={connect}
+              onChange={({ target }) => setConnect(target.checked)}
+            />
+          </Grid>
```

Comment:
Please add newlines in between code blocks, makes it easier to navigate with vim. :)

Reply:
you mean space between `Grid` components?